package.json

```diff
@@ -25,15 +24,19 @@
     "package:osx": "gulp package:osx",
     "package:linux": "gulp build && build --platform linux --arch all -d deb",
     "package:all": "gulp package:all",
-    "prettify": "gulp prettify"
+    "prettify": "gulp prettify",
+    "dist:osx": "build --platform darwin",
+    "dist:linux": "build --platform linux",
+    "dist:win32": "build --platform win32 --arch ia32",
+    "dist:win64": "build --platform win32 --arch x64"
   },
   "devDependencies": {
     "babel-core": "^6.7.5",
     "babel-loader": "^6.2.4",
     "babel-preset-react": "^6.5.0",
     "chromedriver": "^2.20.0",
     "del": "^2.2.0",
-    "electron-builder": "^3.11.0",
+    "electron-builder": "3.20.0",
```

Comment:
Fixed version due to https://github.com/electron-userland/electron-builder/issues/402